docs/admin/node.md

```diff
@@ -247,6 +248,12 @@ Set the `cpu` and `memory` values to the amount of resources you want to reserve
 Place the file in the manifest directory (`--config=DIR` flag of kubelet).  Do this
 on each kubelet where you want to reserve resources.
 
+## API Object
+
+Node is a top-level resource in the kubernetes REST API. More details about the
+API object can be found at: [Node API
+object](https://htmlpreview.github.io/?https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-reference/definitions.html#_v1_node).
```

Comment:
Actually, https://htmlpreview.github.io/?https://github.com/GoogleCloudPlatform/kubernetes/HEAD/docs/api-reference/definitions.html#_v1_node does work. I can link to that on HEAD and update mark-new-version to replace HEAD by the new release version as it does for api/*types.go: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/build/mark-new-version.sh#L112.
@bgrant0607 @thockin does that sound good?